endpoints/openrtb2/sample-requests/amp/valid-supplementary/aliased-buyeruids-camel-case.json

```diff
@@ -0,0 +1,77 @@
+{
+  "description": "Amp request where root.user.ext.prebid.buyeruids field has camel case bidder name",
+  "query": "tag_id=101",
+  "config": {
+    "mockBidders": [
+      {
+        "bidderName": "yahooAds",
+        "currency": "USD",
+        "price": 2
+      }
+    ]
+  },
+  "mockBidRequest": {
+    "id": "request-with-user-ext-obj",
+    "site": {
+      "page": "test.somepage.com"
+    },
+    "imp": [
+      {
+        "id": "my-imp-id",
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 600
+            }
+          ]
+        },
+        "ext": {
+          "prebid": {
+            "bidder": {
+              "YahooAds": {
```

Comment:
Also add a test case for the request that has an alias in the `request.ext.prebid.aliases` field.

Reply:
Added in [b43a4cf](https://github.com/prebid/prebid-server/pull/3152/commits/b43a4cfc1feeba7a0586fafd49890437b083f375)